### DISCUSSION TITLE
Consolidating Software Version Tracking in Nautobot

### DISCUSSION THREAD

**Discussion by digital-fortress** [nautobot/nautobot#6040] [Plugin (App) Ideas & Development]
Nautobot's built-in software version tracking and the nautobot-device-lifecycle-mgmt plugin each have separate entries for tracking software, resulting in redundant information. Is it possible to consolidate this into a single entry, where the plugin retrieves software data from Nautobot's built-in software model?

**Comment by jinesh-kaiprath** [comment]
I see this has been actioned on the new Device Lifecycle version, even though no one acknowledged the request. Lol.
